<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedditObjectSettingsDialog</class>
 <widget class="QWidget" name="RedditObjectSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>999</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QListView" name="reddit_objects_list_view"/>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="info_form">
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Date Created:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Date Added:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Last Download:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="name_label">
             <property name="text">
              <string>name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="id_label">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="date_created_label">
             <property name="text">
              <string>date_created</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="date_added_label">
             <property name="text">
              <string>date_added</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="last_download_label">
             <property name="text">
              <string>last_download</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_27">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Basic Info</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="download_info_form">
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Post Count:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="post_count_label">
             <property name="text">
              <string>post_count</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Comment Count:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="comment_count_label">
             <property name="text">
              <string>comment_count</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Content Count:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="content_count_label">
             <property name="text">
              <string>content_count</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_26">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Download Info</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="post_download_form">
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Post Limit:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Score Limit:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Score Limit Eval:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Avoid Duplicates:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Download Videos:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Download Images:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Sort Posts By:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Download Naming:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="lock_settings_checkbox">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Lock Settings:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="post_limit_spinbox"/>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="score_limit_spinbox"/>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="score_limit_operator_combo"/>
           </item>
           <item row="6" column="1">
            <widget class="QDateTimeEdit" name="dateTimeEdit"/>
           </item>
           <item row="7" column="1">
            <widget class="QCheckBox" name="avoid_duplicates_checkbox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QCheckBox" name="download_videos_checkbox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QCheckBox" name="download_images_checkbox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QComboBox" name="post_sort_combo"/>
           </item>
           <item row="12" column="1">
            <widget class="QComboBox" name="download_naming_combo"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_19">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Post Download Settings</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>NSFW Filter:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QComboBox" name="nsfw_filter_combo"/>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="date_limit_checkbox">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Date Limit:    </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Download Links</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Download:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Comment Limit:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Score Limit:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Score Limit Eval:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_20">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Comment Download Settings</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comment_download_combo"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comment_content_download_combo"/>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="comment_score_operator_combo"/>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="comment_limit_spinbox"/>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="comment_score_limit_spinbox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="reset_button">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="dialog_button_box">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
         </property>
         <property name="centerButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
